lib/Importer/ONNXModelLoader.cpp

```diff
@@ -985,7 +985,7 @@ Error ONNXModelLoader::loadConstant(const ONNX_NAMESPACE::NodeProto &op,
 }
 
 /// Retrieves data from a constant Tensor and stores it in a vector.
-template <typename T>
+template <typename T, typename datatype = ssize_t>
 static void helperSetter(Constant *constT, std::vector<ssize_t> &vec) {
```

Comment:
Didn't you need to also update vec to use `datatype` here? If this isn't causing any build issues not sure why it's necessary in the first place. Was the idea that there could be a troublesome implicit cast on 32-bit architectures, so this was just to be safe? It'd be better if this was actually exposed somehow in a test, though not sure it's easy.

Reply:
As of now, I have updated vec to use datatype to be safer side.